src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +131,39 @@ public String beforeOrAfter(LocalDate someDate) {
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.ofHours(2));
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        Optional<LocalDate> result;
+        try {
+            StringBuilder data = new StringBuilder(date);
+            data.insert(6, "-");
+            data.insert(4, "-");
```

Comment:
cane we use a formatter instead of this hardcode?

Reply:
resolved